src/main/java/ch/njol/skript/expressions/ExprItemWithTooltip.java

```diff
@@ -0,0 +1,75 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.aliases.ItemType;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemFlag;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Item with Tooltip")
+@Description({
+	"Get an item with or without entire/additional tooltip.",
+	"If changing the 'entire' tooltip of an item, nothing will show up when a player hovers over it.",
+	"If changing the 'additional' tooltip, only specific parts (which change per item) will be hidden."
+})
+@Examples({
+	"set {_item with additional tooltip} to diamond with additional tooltip",
+	"set {_item without entire tooltip} to diamond without entire tooltip"
+})
+@RequiredPlugins("Spigot 1.20.5+")
+@Since("INSERT VERSION")
+public class ExprItemWithTooltip extends PropertyExpression<ItemType, ItemType> {
+	static {
+		if (Skript.methodExists(ItemMeta.class, "isHideTooltip")) {// this method was added in the same version as the additional tooltip item flag
+			Skript.registerExpression(ExprItemWithTooltip.class, ItemType.class, ExpressionType.PROPERTY,
+				"%itemtypes% with[:out] [entire|:additional] tool[ ]tip[s]"
+			);
+		}
+	}
+
+	private boolean hide, entire;
+
+	@Override
+	public boolean init(Expression<?>[] expressions, int matchedPattern, Kleenean isDelayed, SkriptParser.ParseResult parseResult) {
+		setExpr((Expression<ItemType>) expressions[0]);
+		hide = parseResult.hasTag("out");
```

Comment:
```suggestion
		without= parseResult.hasTag("out");
```
It would be nice if the name of the variable matched the tag, optional